relayer/pkg/chainlink/ocr2/transmissions_cache.go

```diff
@@ -133,9 +134,8 @@ func (c *transmissionsCache) LatestRoundRequested(
 	c.tdLock.RLock()
 	defer c.tdLock.RUnlock()
 	configDigest = c.transmissionDetails.Digest
-	epoch = c.transmissionDetails.Epoch
-	round = c.transmissionDetails.Round
-	err = c.assertTransmissionsNotStale()
+	epoch = 0
+	round = 0
```

Comment:
the changes look fine - but why is `LatestRoundRequested` implemented in two places? it seems very easy to make changes in one place but forget to make changes in the other

Reply:
originally i think it was to have a cached response. libocr uses the transmissions_cache to satisfy the `ContractTransmitter` interface. 

now that we know we can always return 0 I think we don't need the underlying method. i'll edit that out

Reply:
oh wait, actually the Reader needs to implement it because it must conform to the interface was well

Reply:
i removed the need to query the chain and to use the lock

Reply:
I guess my suggestion is - instead of having zero values implemented twice, why not just call `c.reader.LatestRoundRequested` and add a note saying that this will always return zero values?

that way you only have hard-coded values in one place rather than two places

Reply:
oh i see what you mean. will do 👍